index.js

```diff
@@ -105,35 +116,67 @@ config.rules['declaration-block-properties-order'] = [[
   'min-height',
   'max-height',
   'overflow',
+  'overflow-x',
+  'overflow-y',
+  'resize',
   'list-style',
+  'list-style-type',
+  'list-style-position',
+  'list-style-image',
+  'caption-side',
```

Comment:
The `background-repeat` is not being repeated here, it's being removed from line 92 and repositioned to better match the array represented in the repo you linked me to.